Loaded module: cuda/11.8
ERROR: Unable to locate a modulefile for 'python3/3.9.18'
WARNING: Value for scheme.platlib does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /zhome/4e/b/208805/.local/lib/python3.9/site-packages
sysconfig: /zhome/4e/b/208805/.local/lib64/python3.9/site-packages
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
Traceback (most recent call last):
  File "/dtu/p1/johlau/Thesis-Synthex/radchest-angle.py", line 21, in <module>
    patient = deepdrr.Volume.from_nifti(
  File "/zhome/4e/b/208805/.local/lib/python3.9/site-packages/deepdrr/vol/volume.py", line 508, in from_nifti
    materials = cls.segment_materials(
  File "/zhome/4e/b/208805/.local/lib/python3.9/site-packages/deepdrr/vol/volume.py", line 307, in segment_materials
    materials = cls._segment_materials(hu_values, use_thresholding=use_thresholding)
  File "/zhome/4e/b/208805/.local/lib/python3.9/site-packages/deepdrr/vol/volume.py", line 253, in _segment_materials
    return load_dicom.conv_hu_to_materials(hu_values)
  File "/zhome/4e/b/208805/.local/lib/python3.9/site-packages/deepdrr/load_dicom.py", line 149, in conv_hu_to_materials
    materials = segmentation_network.segment(hu_values)
  File "/zhome/4e/b/208805/.local/lib/python3.9/site-packages/deepdrr/segmentation.py", line 84, in segment
    output_tensor = self.model.forward(
KeyboardInterrupt
Terminated
